egs/wsj/s5/local/rnnlm/tuning/run_lstm_tdnn_1c.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+# Copyright 2012  Johns Hopkins University (author: Daniel Povey)  Tony Robinson
+#           2017  Hainan Xu
+#           2017  Ke Li
+
+# This is the same as 1b, with two exceptions: TDNN is used as the acoustic model,
+# and the baseline is big-dict 4-gram LM.
+
+# (Manually rename chain dirs for lstm_tdnn_1b and lstm_tdnn_1c as 
+# chain1b and chain1c)
+# local/chain/compare_wer.sh exp/chain1b/tdnn1g_sp exp/chain1c/tdnn1g_sp
```

Comment:
I think something is not right here.  You are just comparing two different runs of the tdnn1g model without RNNLM rescoring.  You should talk to a more senior student and have them explain to you how this works.

Reply:
Ok I'll talk to Ke about this.